sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -200,6 +200,38 @@ object SQLConf {
     Map(configs.map { cfg => cfg.key -> cfg } : _*)
   }
 
+  /**
+   * Holds information about keys that have been deprecated.
+   *
+   * @param key The deprecated key.
+   * @param version Version of Spark where key was deprecated.
+   * @param comment Additional info regarding to the removed config. For example,
+   *                reasons of config deprecation, what users should use instead of it.
+   */
+  case class DeprecatedConfig(key: String, version: String, comment: String)
+
+  /**
+   * Maps deprecated SQL config keys to information about the deprecation.
+   *
+   * The extra information is logged as a warning when the SQL config is present
+   * in the user's configuration.
+   */
+  val deprecatedSQLConfigs: Map[String, DeprecatedConfig] = {
+    val configs = Seq(
+      DeprecatedConfig("spark.sql.hive.verifyPartitionPath", "3.0.0",
+        "This config will be replaced by spark.files.ignoreMissingFiles."),
```

Comment:
“will be” -> "is"?

Reply:
Right, it has been already replaced by the config: https://github.com/apache/spark/blob/5264164a67df498b73facae207eda12ee133be7d/sql/hive/src/test/scala/org/apache/spark/sql/hive/QueryPartitionSuite.scala#L79-L84